sync/syncer.go

```diff
@@ -189,24 +195,28 @@ func (s *Syncer) lastTickedLayer() types.LayerID {
 }
 
 func (s *Syncer) Synchronise() {
-	for currentSyncLayer := s.lValidator.ValidatedLayer() + 1; currentSyncLayer <= s.lastTickedLayer(); currentSyncLayer++ {
-		lastTickedLayer := s.lastTickedLayer()
-		s.Info("syncing layer %v current consensus layer is %d", currentSyncLayer, lastTickedLayer)
-
-		if s.IsSynced() && currentSyncLayer == lastTickedLayer {
-			continue
-		}
-
+	defer s.syncFuncWg.Done()
```

Comment:
SyncRoutine/FuncWg, the original name is not that great either but lets keep it consistent (feel free to invent a new name )